lib/rubygoal/player.rb

```diff
@@ -11,15 +11,17 @@ class Player
 
     include Moveable
 
-    attr_reader :side, :type
+    attr_reader :side, :type, :name
     attr_accessor :coach_defined_position
 
-    def initialize(game, side)
+    def initialize(game, side, name)
       super()
 
       @time_to_kick_again = 0
       @side = side
       @player_movement = PlayerMovement.new(game, self)
+      @game = game
```

Comment:
This attribute is unused and should be removed

Reply:
It wiil be removed.